docs/SlavaUkraini/examples/accessibility/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: tutorial_v2
+title: A guide to basic Leaflet accessibility
+---
+
+## Accessible maps
+
+Web accessibility is the inclusive practice of ensuring no barriers exist
+that would prevent interactions or information access.
+
+This guide to Leaflet accessibility can help you create maps that are usable
+to persons of a wide range of abilities.
+
+### Preserve useful defaults
+
+Leaflet comes with a set of useful defaults.
+
+The map container and markers are keyboard operable by default,
+this enables users who are unable to use a pointing device.
+Consider the effects on your users before changing defaults such as these.
+
+### Markers must be labelled
+
+When using markers,
+it is vital to ensure each has a unique and descriptive
+[`alt`](/reference.html#marker-alt)
+or
+[`title`](/reference.html#marker-alt):
+
+<pre><code>var marker = L.marker([50.4501, 30.5234],
+  {alt: 'Kyiv'}).addTo(map) // "Kyiv" is the <a href="https://www.w3.org/TR/accname-1.1/#dfn-accessible-name"><em>accessible name</em></a> of this marker
+  .bindPopup('Kyiv, Ukraine is the birthplace of Leaflet!');</code></pre>
+
+Resulting in markers that are discernible to
+[screen reader](https://en.wikipedia.org/wiki/Screen_reader)
+users:
+
+{% include frame.html url="example.html" width=600 height=400 %}
+
+In the case of `divIcon`s,
+[custom HTML](reference.html#divicon-html)
+can otherwise provide a visual or non-visual label.
+
+### Test your maps
+
+The best way to discover accessibility issues
+is to test your maps using only a keyboard,
+as well as using a screen reader.
+You may already have a screen reader pre-installed,
+for example:
+
+- Narrator on Windows
```

Comment:
Links, links, links. Don't tell me the name of the tool, point me to its documentation. How do I enable it?

Also, whta about desktop linux screenreaders? A reference+link to Orca, maybe?

Reply:
Totally agree. I've added links that should be useful.

Do you think there should be explicit instructions in this section? Of course it'd be useful for readers to understand that avid screen reader users use different keyboard shortcuts to navigate documents and not only <kbd>tab</kbd>, e.g. navigate by headings, read text in a certain direction using arrow keys, pause speech on <kbd>ctrl</kbd> (very useful for devs who aren't used to the constant chatter), etc.

The docs for each screen reader explain how to enable (and disable) the screen readers, and the keyboard shortcuts.

Reply:
I'd say "no" to explicit instructions. Links are enough IMHO.